src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/Metadata/ModelMetadataIdentity.cs

```diff
@@ -66,17 +66,37 @@ public static ModelMetadataIdentity ForType(Type modelType)
             };
         }
 
+        /// <summary>
+        /// Creates a <see cref="ModelMetadataIdentity"/> for the provided parameter.
+        /// </summary>
+        /// <param name="parameter">The parameter type.</param>
```

Comment:
One of "parameter information" or `<see cref="ParameterInfo"/>`. Same for next overload.